.github/workflows/ci.yml

```diff
@@ -164,10 +164,15 @@ jobs:
         path: ./build/Testing/Temporary
         name: f3d-tests-artifact-${{matrix.os}}-${{matrix.vtk_version}}
 
-    - name: Install
+    - name: Install (ALL)
       working-directory: ${{github.workspace}}/build
       run: cmake --install .
 
+    - name: Install (mimetypes on Linux)
+      if: matrix.os == 'ubuntu-latest'
+      working-directory: ${{github.workspace}}/build
+      run: cmake -DCOMPONENT=mimetypes -P cmake_install.cmake
```

Comment:
I'm a bit confused by this. How will compiling user find that they need to do that ?

Reply:
We cannot really use an option I think because a plugin can be built externally.  
So I added `EXCLUDE_FROM_ALL` because we don't want the mime-type XML always installed, right?  
So I found this component-specific installation trick.  

However, looking at the comments from [this answer](https://stackoverflow.com/a/9192877/2609654), it seems like we can replace `cmake -DCOMPONENT=mimetypes -P cmake_install.cmake` by `cmake --install . --component mimetypes` which looks way better.  
I also think we can generalize this usage to more components, by installing optional component explicitly instead of having CMake variables that configure what's installed in `ALL`.

Reply:
Got it. Indeed, that is much better looking an generally acceptable. Make sure to document it thoroughly in build.md. We will need to inform packagers.